handlers/handler-bizkeeper/src/main/java/org/apache/servicecomb/bizkeeper/FallbackPolicy.java

```diff
@@ -22,7 +22,7 @@
 public interface FallbackPolicy {
   String name();
 
-  Response getFallbackResponse(Invocation invocation);
+  Response getFallbackResponse(Invocation invocation, Throwable error);
```

Comment:
seems not compatible?

Reply:
I think the change is fine. Users can change their implementations easily.